Modules/_testinternalcapi.c

```diff
@@ -12,7 +12,7 @@
 #define PY_SSIZE_T_CLEAN
 
 #include "Python.h"
-#include "pycore_byteswap.h"     // _Py_bswap32()
+#include "pycore_bitutils.h"     // _Py_bswap32()
```

Comment:
Should the comment on this line be changed?

Reply:
I don't try to list all functions used by an include, but just one to explain why the include is needed. I use that information to check sometimes if an include can be removed.